server.js

```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+var express = require('express');
```

Comment:
Hm, I feel hesitant merging this change in just because Heroku isn't properly configured to run the `start.js` script before running `server.jsx`, I think figuring that out would be a better change since it wouldn't require modifying application code.